src/main/docs/guide/modules-databases.adoc

```diff
@@ -44,3 +44,18 @@ test-resources:
 ----
 
 ====
+
+For a complete list configurable properties refer to the https://micronaut-projects.github.io/micronaut-test-resources/latest/guide/configurationreference.html#io.micronaut.testresources.testcontainers.TestContainersConfiguration[Configuration Reference for Test Containers].
+
+[NOTE]
+====
+The initial image download of some database images can take some time depending the size and speed of your internet connection. In this case you may receive a `ReadTimeoutException`, which requires configuring the startup timeout, for example the following extends the startup time for the `oracle` container image: 
```

Comment:
While this is sufficient if you use the embedded test resources module, most users will instead use the test resources client via the Gradle or Maven plugins, in which case you also need to set the client timeout: https://micronaut-projects.github.io/micronaut-gradle-plugin/latest/#_configuring_the_test_resources_plugin

Reply:
think we need to clarify this as well somewhere

Reply:
Note that even when I had a slow internet connection, I never had to change the `startup-time`, but I did have to change the client timeout.

Reply:
I reworded this and added links